### PR TITLE
Update extension: New command

### PR DESCRIPTION
**Pull request by pradeepb28**
## Description

The purpose of this command is to retrieve list of items from an app. 

Here is how it works:
- The app saves data under defaults write .
- The Raycast extension picks up the data from that defaults write and show it in the list.

Defaults write is a way of communicating from app to extension without any servers.

## Screencast
CleanShot 2022-06-22 at 18 43 16@2x

CleanShot 2022-06-22 at 18 42 59@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/spaces/CHANGELOG.md [hunk 1]
```diff
@@ -2,5 +2,13 @@
...
 ## [Initial Version] - 2022-06-21
 
-If you have the Spaces app, you can access it straight from Raycast. This release contains one command:
-Create: The command opens a windows to create a new space in the app.
+If you have the Spaces app, you can access it straight from Raycast.
+This release contains one command:
+
+- Create Space: The command opens a windows to create a new space in the app.
+
+## 1.1 - 2022-6-22
```
Comment (danielsequeira) [inline extensions/spaces/CHANGELOG.md:10]: To keep coherent and follow our changelog convention, you could replace 1.1 with [New Additions]
```suggestion
## [New Additions] - 2022-6-22
```

**Review by danielsequeira — COMMENTED** [review]
Thank you for the PR @pradeepb28.

Everything looks good, I just added a small comment.

Outside of the scope of this PR, I noticed that your extension doesn't have a metadata folder with screenshots of the extension. It would be nice to eventually create a metadata folder for your extension and place its screenshots there.

**Comment by pradeepb28** [comment]
> 

added meta folder w. screenshots and for some reason my cleanshot draws a line on top of image and I don't have a way to fix it other than reporting the bug to cleanshot team.

**Comment by danielsequeira** [comment]
@pradeepb28 weird, I never experienced this one. In this case let's not add the screenshots just yet, I would remove them from the PR and add them once you figured out what is going on with cleanshot and fixed it

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**